packages/forms-web-app/__tests__/unit/routes/householder-planning/eligibility/index.test.js

```diff
@@ -14,9 +14,9 @@ describe('routes/householder-planning/eligibility/index', () => {
 
   it('should define the expected routes', () => {
     expect(use).toHaveBeenCalledWith(listedBuildingHouseholderRouter);
-    expect(use).toHaveBeenCalledWith(enforcementNoticeHouseholderRouter);
-    expect(use).toHaveBeenCalledWith(grantedOrRefusedHouseholderRouter);
+    expect(use).toHaveBeenCalledWith(dateDecisionDueHouseholderRouter);
+    expect(use).toHaveBeenCalledWith(grantedOrRefusedRouter);
 
-    expect(use.mock.calls.length).toBe(3);
+    expect(use.mock.calls.length).toBe(4);
```

Comment:
Where's the fourth router?

Reply:
listed-building, granted-or-refused, enforcement-notice, date-decision-due